### PR TITLE
Replaced username writing with a proper dropdown selection

### PR DESCRIPTION
**Pull request by PibePlayer**
It's saved to a file named names.txt and ordered by last used.
"Fixes" #20

### CODE REVIEW

**Comment by PibePlayer** [comment]
@AfoninZ @Dakkaron @Sebastian-byte is anyone able to review/merge here?

**Comment by PibePlayer** [comment]
@AfoninZ @Dakkaron @Sebastian-byte is anyone able to review/merge here?

**Comment by Sebastian-byte** [comment]
Seems ok for me, I will wait for @AfoninZ thought.

File: application/LaunchController.cpp [hunk 1]
```diff
@@ -37,15 +37,68 @@ void LaunchController::login()
...
+    //nameSelector.setOption(QInputDialog::UseListViewForComboBoxItems); //In case we want to use a list instead of a combobox
+    nameSelector.setComboBoxItems(names);
+    nameSelector.setComboBoxEditable(true);
+    nameSelector.setWindowTitle("Select Username...");
+
+    if(nameSelector.exec() == QDialog::Accepted){
+        name = nameSelector.textValue();
+        if(name != names[0]){
+            namesFile.open(QIODevice::WriteOnly | QIODevice::Text);
+            namesFile.write(name.toStdString().c_str());
```
Comment (PibePlayer) [inline application/LaunchController.cpp:75]: I could do it in a more QT like way replacing this line with:
`QTextStream writer(&namesFile); writer << name.toUtf8();`

Comment (Sebastian-byte) [reply application/LaunchController.cpp:75]: Yes, do that, also can you fix the indent problems that your code has?

Comment (Sebastian-byte) [reply application/LaunchController.cpp:75]: For example:
```c
if(...){
...
}

// Should be

if (...) {
...
}
```

Comment (PibePlayer) [reply application/LaunchController.cpp:75]: Allright, doing that rn

Comment (PibePlayer) [reply application/LaunchController.cpp]: Done

**Comment by PibePlayer** [comment]
Is @AfoninZ active?

**Comment by Sebastian-byte** [comment]
I think it doesn't have time to check this PR.

**Comment by PibePlayer** [comment]
@Sebastian-byte until he comes back I'll maintain my fork which has this included and I just added Continuous Integration so new features can be tested on the fly ;) [PibePlayer/MultiMC5-Cracked](https://github.com/PibePlayer/MultiMC5-Cracked)

**Comment by Sebastian-byte** [comment]
@PibePlayer Your CI doesn't seems to work, I just get 0xC00007B error.

**Comment by PibePlayer** [comment]
@Sebastian-byte please bring that issue to my repo so I check it out (I don't have a computer rn), and by the way tell me when does that appear? Opening the app? Downloading an Instance? Opening one?

**Comment by PibePlayer** [comment]
@Sebastian-byte I don't seem to able to reproduce your issue, downloaded the lastest build (8), initialized, downloaded 1.16.5 and played for 5 minutes without problems. What build did you download and how did you get that error?

**Comment by Sebastian-byte** [comment]
Last build used (8)

**Comment by Sebastian-byte** [comment]
I used the last build (8) in a Windows 10 64-bit operating system

**Comment by Sebastian-byte** [comment]
error

**Comment by Sebastian-byte** [comment]
I think I found the problem, "VCRUNTIME140.dll" is 64-bit, it should be 32-bit since the executable is, this is caused by 64-bit .NET framework in the build system.
image

**Comment by PibePlayer** [comment]
Hmm... I'm gonna check it out, for now I added the OpenSSL dlls that weren´t installed

**Comment by PibePlayer** [comment]
Finally added issues tab to my repo

**Comment by Sebastian-byte** [comment]
You can use this zlib, is the correct one: https://downloads.sourceforge.net/project/gnuwin32/zlib/1.2.3/zlib-1.2.3-bin.zip

**Comment by PibePlayer** [comment]
[develop.yml](https://github.com/PibePlayer/MultiMC5-Cracked/blob/develop/.github/workflows/develop.yml#L101) I'm using that one

**Comment by Sebastian-byte** [comment]
Why do you download zlib-1.2.3-lib?

**Comment by AfoninZ** [comment]
wait what's happened

**Comment by AfoninZ** [comment]
ok I think I've reverted it bc idk if it compiles or not
@Sebastian-byte thanks for keeping an eye on this repo, I just really can't do any maintaining for the next couple of months, so no need for my approval for anything - if it seems ok to you, it probably seems ok to me. Maybe I'll get back to maintaining the repo, but not now

**Comment by PibePlayer** [comment]
> wait what's happened

What do you mean? It compiles just right with this one, I had problems on my Github Actions Implementation, which @Sebastian-byte copied and corrected, it wasn't a problem from my commits 😅

**Pull request merged**